src/binary/coreir.cpp

```diff
@@ -171,8 +188,12 @@ int main(int argc, char *argv[]) {
     cout << "Running vpasses" << endl;
 
     auto vpass = static_cast<Passes::Verilog*>(c->getPassManager()->getAnalysisPass("verilog"));
-    
-    vpass->writeToStream(*sout);
+
+    if (split_files) {
```

Comment:
@rdaly525 right now this flag is only considered in verilog mode. Other modes will ignore it altogether. What would you like to do here. I can add the stub method `writeToFiles()` for each of the other passes (where the implementation is to raise a not-implemented error. Otherwise we can leave it as is and document that this only works for verilog mode.